snapshot-labs/stamp
# fix: ignore starknet default avatar image

Fix #257 

Ignore starknet default image on avatar resolver

### Test 

- Go to https://snapshot.box/#/sn:0x07c251045154318a2376a3bb65be47d3c90df1740d8e35c9b9d943aa3f240e50/proposal/1/votes
- Select one address with default avatar
- Go to http://localhost:3008/avatar/ADDRESS with the copied address
- It should return blockicon avatar
- Do the same with an address with a non-default avatar
- It should return the avatar
